components/calendar/Header.tsx

```diff
@@ -16,6 +39,7 @@ export interface HeaderProps {
   onTypeChange?: (type: string) => void;
   value: any;
   validRange?: [moment.Moment, moment.Moment];
+  renderHeader?: (returnData: RenderHeader) => React.ReactNode;
```

Comment:
Our naming standard is `xxxRender` => `headerRender`

Reply:
Thanks for comment. 

I get this example from https://ant.design/components/date-picker/#WeekPicker 

in WeekPicker we have `renderExtraFooter` prop. But i will fix it

Thanks

Reply:
Yes.  We have some render props miss the naming standard. `renderExtraFooter` is the one 😄